lib/Runtime/ByteCode/OpCodes.h

```diff
@@ -834,6 +834,7 @@ MACRO_EXTEND_WMS(       StPropIdArrFromVar, ElementSlot,    OpSideEffect|OpHasIm
 MACRO_EXTEND_WMS(       Restify,            Reg4,           OpSideEffect|OpHasImplicitCall)
 MACRO_EXTEND_WMS(       NewPropIdArrForCompProps, Reg1Unsigned1, OpSideEffect)
 
+MACRO_BACKEND_ONLY(BigIntLiteral, Empty, None) // Load BigInt literal
```

Comment:
It looks like this is only used for debug output. Rather than adding a new opcode, how about just using the string "BigIntLiteral" directly?

Reply:
Never mind, we'll probably need this later.

---
In reply to: [218171807](https://github.com/Microsoft/ChakraCore/pull/5706#discussion_r218171807) [](ancestors = 218171807)